benchmarks/dynamo/torchbench.py

```diff
@@ -85,6 +85,15 @@ def setup_torchbench_cwd():
     "clip",
 }
 
+SKIP_DUE_TO_CONTROL_FLOW = {
```

Comment:
Any links for documenting those errors? It will be nice to have centralized location for the information.